draft-packages/popover/KaizenDraft/Popover/styles.scss

```diff
@@ -112,25 +109,32 @@ $negative-box-border-color: $color-red-300;
 }
 
 .header {
-  @include kz-typography-heading-6;
-  @include ca-inherit-baseline;
-  padding: ($ca-grid / 4) ($ca-grid / 2);
-  margin-bottom: $ca-grid / 4;
+  font-family: $typography-paragraph-body-font-family;
+  font-size: $typography-heading-6-font-size;
+  font-weight: $typography-heading-6-font-weight;
+  letter-spacing: $typography-heading-6-letter-spacing;
+  line-height: $typography-paragraph-small-line-height;
+  position: static;
+  margin-bottom: $spacing-xs;
   display: flex;
   align-items: center;
   white-space: nowrap;
+  padding-right: $spacing-md;
 }
 
 .container {
-  @include ca-inherit-baseline;
-  @include kz-typography-paragraph-small;
-  padding: ($ca-grid / 6) ($ca-grid / 2) ($ca-grid);
+  position: static;
+  font-family: $typography-paragraph-body-font-family;
+  font-size: $typography-heading-6-font-size;
+  font-weight: $typography-paragraph-small-font-weight;
+  letter-spacing: $typography-paragraph-small-letter-spacing;
+  line-height: $typography-paragraph-small-line-height;
```

Comment:
Our type styles should be the one variable across the whole declaration. Guessing that first one is all `heading-6` and the second is all `paragraph-small`. If it looks incorrect in the browser we'll need to flag this in the UI Kit.